### DISCUSSION TITLE
Obrazki/ikony kart jako svg, a djangowy ImageField

### DISCUSSION THREAD

**Discussion by itchystone** [emkarcinos/WMIAdventure#440] [Codding]
Chyba nie gadaliśmy jeszcze o tym jakie mają być obrazki/ikony kart, więc nie wiem czy chieliśmy wogóle obsługiwać pliki svg. Jeśli tak to djangowy `ImageField` nie przyjmuje plików svg i możemy albo z nich zrezygnować, albo zmienić pole `image` na np. `FileField`, albo na `SVGAndImageFormField`. Więcej o tym znalazłem [tutaj](https://stackoverflow.com/questions/38006200/allow-svg-files-to-be-uploaded-to-imagefield-via-django-admin)

**Comment by emkarcinos** [comment]
A jak to wygląda od strony frontendu? Czy wyświetlanie SVG jest bardziej problematyczne niż PNG? Raczej dobrze byłoby żeby to jednak było to SVG, mamy wtedy dobrą skalowalność zapewnianą. Ale musimy też obsłużyć inne formaty, na przykład wspomniany PNG.

**Reply by itchystone** [reply]
Chyba svg najlepsze, chociaż ktoś kto bardziej w frontend umie musiałby potwierdzić

**Reply by emkarcinos** [reply]
Tak, tylko chodzi o to czy powinniśmy ograniczać użytkowników tylko do tego formatu.
To mogłoby być dla nich problematyczne, niektóre rzeczy mogą być trudne do zrobienia wektorowo, nie mówiąc już o znajdowaniu takich assetów np w google grafika. To nie będą jakieś ikonki, tylko full obrazki.

**Reply by mattyl006** [reply]
Nie ma problemów z svg, ja do wszystkiego poza zdjęciami używam svg robiąc stronki
